src/platforms/python/common/performance/instrumentation/custom-instrumentation.mdx

```diff
@@ -106,6 +106,35 @@ def eat_slice(slice):
     swallow()
 ```
 
+## Define Span Creation in a Central Place
+
+If you do not want to have custom performance instrumentation code scattered all over your code base, you can pass a parameter <PlatformIdentifier name="functions-to-trace" /> to your `sentry_sdk.init()` call.
+
+```python
+import sentry_sdk
+
+functions_to_trace = [
+    "myrootmodule.eat_slice",
+    "myrootmodule.swallow",
+    "myrootmodule.chew",
+    "myrootmodule.someothermodule.another.some_function",
+    "myrootmodule.SomePizzaClass.some_method",
+]
+
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+    functions_to_trace=functions_to_trace,
+)
+```
+
+Now whenever a function specified in `functions_to_trace` is executed a span is created and attached as child to the currently running span.
+
+<Alert level="warning" title="Important">
+
+The SDK needs to load the modules of the functions specified in `functions_to_trace` in order to enable performance monitoring for those functions. There can be code in modules that is executing during loading of the module. Please be aware of this. If you do not want this to happen you can use the ways described above to trace your functions.
```

Comment:
```suggestion
To enable performance monitoring for the functions specified in `functions_to_trace`, the SDK needs to load the function modules. Be aware, there may be code being executed in modules during module loading. To avoid this, use the method described above to trace your functions.
```